src/main/java/com/ft/methodeimagemodelmapper/service/GraphicResolver.java

```diff
@@ -0,0 +1,50 @@
+package com.ft.methodeimagemodelmapper.service;
+
+import com.ft.methodeimagemodelmapper.model.EomFile;
+import com.googlecode.pngtastic.core.PngException;
+import com.googlecode.pngtastic.core.PngImage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedInputStream;
+import java.io.ByteArrayInputStream;
+import java.nio.charset.Charset;
+import java.util.Optional;
+
+import static com.ft.methodeimagemodelmapper.service.MethodeImageModelMapper.IMAGE_TYPE;
+
+public class GraphicResolver {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GraphicResolver.class);
+    private static final Charset LATIN_1 = Charset.forName("ISO-8859-1");
+    private static final String TEXT_CHUNK = "tEXt";
+    private static final String GRAPHIC_MEDIATYPE = MethodeImageModelMapper.MEDIATYPE_PREFIX + "png";
+    private static final String GRAPHIC_TYPE = "Graphic";
+    private static final String SOURCE_GRAPHIC_KEY = "FTImageType";
```

Comment:
since we never use the `SOURCE_GRAPHIC_KEY` let's get rid of it and have `SOURCE_GRAPHIC_KEY_LOWERCASE = "FTImageType".toLowerCase()`